model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -329,17 +337,50 @@ def _parse_perf_throughput(self, section):
         raise TritonModelAnalyzerException(
             'perf_analyzer output did not contain throughput.')
 
-    def _parse_perf_latency(self, section):
+    def _parse_perf_latency_avg(self, section):
+        """
+        Parses avg latency from the perf output
+        """
+        try:
+            return self._parse_perf_latency(section, "Avg latency",
+                                            PerfLatencyAvg)
+        except TritonModelAnalyzerException as e:
+            logging.error(str(e) + " -> 0.0 is used instead!")
+            return PerfLatencyAvg(value=0.0)
+
+    def _parse_perf_latency_p90(self, section):
+        """
+        Parses p90 latency from the perf output
+        """
+        return self._parse_perf_latency(section, "p90 latency", PerfLatencyP90)
+
+    def _parse_perf_latency_p95(self, section):
+        """
+        Parses p95 latency from the perf output
+        """
+        return self._parse_perf_latency(section, "p95 latency", PerfLatencyP95)
+
+    def _parse_perf_latency_p99(self, section):
         """
         Parses p99 latency from the perf output
         """
+        return self._parse_perf_latency(section, "p99 latency", PerfLatencyP99)
 
-        p99_latency = re.search('p99 latency: (\d+\.\d+|\d+)', section)
-        if p99_latency:
-            p99_latency = float(p99_latency.group(1)) / 1e3
-            return PerfLatency(value=p99_latency)
+    def _parse_perf_latency(self, section, latency_name=None, function=None):
+        """
+        Parses latency from the perf output
+        """
+        if latency_name == None and function == None:
+            # Use p99 latency as default
+            latency_name = "p99 latency"
+            function = PerfLatency  # Differ from _parse_perf_latency_p99()
```

Comment:
Can you move the comment to the top of the Line 376?

Reply:
Updated.